tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/BlockingRpcClient.java

```diff
@@ -188,10 +187,12 @@ protected void channelRead0(ChannelHandlerContext ctx, RpcResponse rpcResponse)
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause)
         throws Exception {
+      /* Current requests will be failed */
       for(ProtoCallFuture callback: requests.values()) {
         callback.setFailed(cause.getMessage(), cause);
       }
-      
+      requests.clear();
```

Comment:
This case is fatal error and  It will be internal error.
All request is set to fail and then requests should be removed.